tasks/install.yml

```diff
@@ -0,0 +1,151 @@
+---
+- name: download file from a file path
+  get_url:
+    url: https://download.nextcloud.com/server/releases/nextcloud-11.0.3.tar.bz2
```

Comment:
Softcode & Move to the control machine - we might not have internet on the server